galaxy.yml

```diff
@@ -8,7 +8,7 @@ namespace: community
 name: sap_launchpad
 
 # The version of the collection. Must be compatible with semantic versioning
-version: 1.0.0
+version: 1.1.0
```

Comment:
we should removed this changed, after this pr is merged a additional pr will then change the galaxy version and other things.